source/create-file.js

```diff
@@ -14,25 +14,30 @@ module.exports = (
     fileName,
     outputPath,
     prettierOptions,
-    searchPath
+    searchPath,
+    fileExtension
   }
 ) => {
   assert(searchPath, 'Options.searchPath is required.');
   assert(outputPath, 'Options.outputPath is required.');
   assert(fileName, 'Options.fileName is required.');
-
+  fileExtension = fileExtension ? '.' + fileExtension.trim('.') : '.jsx';
```

Comment:
Er denne linja nødvendig egentlig? Tror du bare kan ha 'fileExtension = ".jsx"' i parameteret

Reply:
Ja, i teorien kan jeg det. Jeg la til det der så folk kan både skrive fileExtension="tsx" og fileExtension=".tsx". Men jeg kan vel bare skrive i readMe hvordan man forventer å få extension.

Reply:
Åja, skjønner! Men .trim() fjerner bare whitespaces, du kan bruke .replace(".", "") for å fjerne punktumet

Reply:
Åja, sant det. Er så vant til strip fra Python. Da bare fjerner jeg hele linja. Putter defaulten i parameteret, og skriver at vi forventer ".[fileExtension]" i README.